EXAMPLES.md

```diff
@@ -512,39 +512,145 @@ Stylus Battery Strength: 77%
+| 2 | KP+ |   KP9    |    Fn    |      |       e       | Vol. Down F2 |g  |h  |     \       |        |Bright. Down F7  |       KP8         |       |     -         |Scan Code e016  |    Right       |
+| 3 | KP2 |  LMeta   |   Tab    |      | Audio Prev F4 |   Mute F1    |t  |y  |      o      |        |  Audio Next F6  |    Project F9     |       | Framework F12 |      End       | Scan Code e01a |
+| 4 | KP3 |   KP7    |    `     |      |  Caps Lock    |      s       |5  |6  | RF Kill F10 |        |  Play Pause F5  |      Ro Kana      |       |       0       |    + =         |                |
+| 5 | KP. |   Home   |    1     |      |       3       |      2       |4  |7  |      9      |        |        8        |       102nd       |       |       p       |       BS       |      KP4       |
+| 6 | KP1 | Page Up  | Muhenkan |      |  Vol. Up F3   |      w       |r  |u  | PrtScr F11  |        |        i        |       Left        |       |      [        |      ]         |    KP5         |
+| 7 | KP/ | Num Lock |    a     |      |   Page Down   |    Escape    |f  |j  |      l      |        |        k        |       Menu        |       |       ;       |       d        |      KP6       |
+
+> Matrix w/ scancodes
 
-### Framework 16
+|    |   0    |   1    |   2    |   3    |   4    |   5    |   6    |   7    |   8    |   9    |   10   |   11   |   12   |   13   |   14   |   15   |
+| ---|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|--------|
+|  0 | 0x0021 | 0xe071 | 0x0015 | 0xe011 | 0xe05a | 0x0022 | 0x002A | 0x003a | 0x0049 | 0x0059 | 0x0041 | 0x0013 | 0xe014 | 0x004a | 0x0052 | 0x006a |
+|  1 | 0x007B | 0xe070 | 0x0070 | 0x0011 | 0x0029 | 0x001a | 0x0032 | 0x0031 | 0xe072 | 0x0012 | 0x007c | 0x0064 | 0x0014 | 0xe075 | 0x005a | 0x000a |
+|  2 | 0x0079 | 0x007D | 0x00ff | 0x0000 | 0x0024 | 0x0006 | 0x0034 | 0x0033 | 0x005d | 0x0000 | 0x0083 | 0x0075 | 0x0000 | 0x004e | 0xe03c | 0xe074 |
+|  3 | 0x0072 | 0xe01f | 0x000D | 0x0000 | 0x000c | 0x0005 | 0x002c | 0x0035 | 0x0044 | 0x0000 | 0x000b | 0x0001 | 0x0000 | 0x0007 | 0xe069 | 0xe054 |
+|  4 | 0x007A | 0x006c | 0x000E | 0x0000 | 0x0058 | 0x001b | 0x002e | 0x0036 | 0x0009 | 0x0000 | 0x0003 | 0x0051 | 0x0000 | 0x0045 | 0x0055 | 0x0000 |
+|  5 | 0x0071 | 0xe06c | 0x0016 | 0x0000 | 0x0026 | 0x001e | 0x0025 | 0x003d | 0x0046 | 0x0000 | 0x003e | 0x0061 | 0x0000 | 0x004d | 0x0066 | 0x006b |
+|  6 | 0x0069 | 0xe07d | 0x0067 | 0x0000 | 0x0004 | 0x001d | 0x002d | 0x003c | 0x0078 | 0x0000 | 0x0043 | 0xe06b | 0x0000 | 0x0054 | 0x005b | 0x0073 |
+|  7 | 0xe04A | 0x0077 | 0x001c | 0x0000 | 0xe07a | 0x0076 | 0x002b | 0x003b | 0x004b | 0x0000 | 0x0042 | 0xe02f | 0x0000 | 0x004c | 0x0023 | 0x0074 |
+
+##### FrameWork 12
+
+If anyone can find the source code with the FW12 matrix please let me know and I'll add info about that to this page. I don't have an FW12 so all my testing was done on an FW13.
```

Comment:
I published it here, let's just link to it.
I think it's more useful to present the table with entries that are human readable, so the reader can know which key is in which row/col.
https://github.com/FrameworkComputer/Framework-Laptop-12/tree/main/InputCover#keyboard-matrix